### DISCUSSION TITLE
New Potion For Finding Bees

### DISCUSSION THREAD

**Discussion by Jayfeather5665** [Resourceful-Bees/ResourcefulBees#171] [Suggestions]
I thought it would bee nice to have a potion that highlights bees in the near by area kind of like the glowing potion effect because they can be hard to find in forests sometimes. It doesn't need to have that big of a range just something to help find them.

**Comment by Epic428** [comment]
Yeah several people have asked about that feature in some manner. I do plan on implementing something I just haven’t decided the details of it yet.

**Reply by MyconidCoven** [reply]
Something like a Beedar (Radar) where you can pick a Bee from a dropdown and it highlights that specific entity only could be pretty useful/cool. Similar to how a Nature's Compass works for finding biomes but with a sight potion type effect built in.
